audit/cmd/notifications/main.go

```diff
@@ -4,11 +4,11 @@ import (
 	"os"
 
 	"github.com/owncloud/ocis/audit/pkg/command"
-	"github.com/owncloud/ocis/audit/pkg/config"
+	"github.com/owncloud/ocis/idm/pkg/config/defaults"
```

Comment:
This should be `audit` not `idm`, right?

Reply:
:+1: thanks, I should not do to many things in parallel...